src/joyride/sci.cljs

```diff
@@ -83,6 +84,7 @@
               :namespaces {'clojure.core {'IFn (sci/copy-var IFn core-namespace)}
                            'clojure.zip zip-namespace
                            'cljs.test cljs-test-config/cljs-test-namespace
+                           'cljs.pprint cljs-pprint-config/cljs-pprint-namespace
```

Comment:
It would be nice to measure the startup difference when using the previous vsix vs this one. E.g. in nbb pprint is lazily loaded. pprint is pretty big, like 200kb of JS or so.

Reply:
I'll have a check.